### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by fusionpot** [Jitnaught/JNSoundboard#16]
so i have been trying to get this to work while using steam and im not having much luck. is there something i am missing or is it no longer able to work while using steam?

**Comment by Jitnaught** [comment]
Works fine for me. Which part is not working?

**Comment by fusionpot** [comment]
well are do you have teamspeak? would like to be able to chat in real time to see whats up

**Comment by Jitnaught** [comment]
No.

**Comment by fusionpot** [comment]
alright, well i set it up like your read me file says. i set steam to use VAC 1 set the playback to VAC1 set the loopback to my headset and mic.

now when i play a file it will play it into game or any other program set to use VAC1 for sound, but i lose the ability to speak in game with my microphone. is this normal or am i missing something?

**Comment by Jitnaught** [comment]
So you're saying that when you talk into your microphone, it's not being heard from the VAC? Or is the problem that you are unable to press the in-game hotkey for the microphone?

**Comment by fusionpot** [comment]
not able to press the hotkey to try and talk when ingame. but i have also tried to use it on teamspeak with no luck with ptt or voice activation

**Comment by Jitnaught** [comment]
Do you have the "Auto press push to talk key" enabled? Have you tried it with that disabled?

**Comment by fusionpot** [comment]
did not have it enabled, just not turned it on trying to work on it still having some issues.

**Comment by Jitnaught** [comment]
When the program is closed are you able to press the hotkey?

**Comment by fusionpot** [comment]
ill check to verify

**Comment by fusionpot** [comment]
ok with teamspeak i am able to use my mic and play the audio board but the autp press talk key does not seem to be working

**Comment by Jitnaught** [comment]
Do you have it set to the correct window and key?

**Comment by fusionpot** [comment]
yes. teamspeak 3 and using R which is the same key used in teamspeak

**Comment by Jitnaught** [comment]
Have you tried it with other applications? Perhaps JN Soundboard is just incompatible with Teamspeak.

**Comment by fusionpot** [comment]
ill try it in game and see if it works with the auto hotkey

**Comment by fusionpot** [comment]
tried with counter strike source. does not work. will try another game.

**Comment by fusionpot** [comment]
just tried in player unknowns battlegrounds. works if i manually hold the talk button but does not auto Que up the talk key

**Comment by Jitnaught** [comment]
Hm, weird. Do you happen to have Garry's Mod installed? That's the main game I tried it on, and I know for sure it worked for me on that game.

**Comment by fusionpot** [comment]
no i don't. also is there any chance with a new update that there can be a option to give custom names instead of seeing the entire sound location for the name.

**Comment by fusionpot** [comment]
like this one also maybe the ability to edit the sound file so we can play the specific part only instead of having to edit all the files first  http://forum.teamspeak.com/threads/125318-RELEASE-RP-Soundboard

**Comment by Jitnaught** [comment]
I'm not sure what to tell you then. I can't test any of the games/applications you've mentioned, so I don't know if it's a problem on your end or the soundboard's end. I guess if you figure it out tell me, otherwise I'll try looking into the problem soon. Sorry man. :/

If you want to make requests I would suggest making a new Issue for each one, that way if I add them I can do so individually (and you'll get notifications about it).
